src/audio_core/sink_context.cpp

```diff
@@ -15,10 +15,17 @@ std::size_t SinkContext::GetCount() const {
 void SinkContext::UpdateMainSink(const SinkInfo::InParams& in) {
     ASSERT(in.type == SinkTypes::Device);
 
-    has_downmix_coefs = in.device.down_matrix_enabled;
-    if (has_downmix_coefs) {
+    if (in.device.down_matrix_enabled) {
         downmix_coefficients = in.device.down_matrix_coef;
+    } else {
+        downmix_coefficients = {
+            1.0f,   // front
+            0.707f, // center
+            0.0f,   // lfe
+            0.707f, // back
+        };
```

Comment:
```suggestion
        downmix_coefficients = {
            0.3694f,   // front
            0.2612f, // center
            0.0f,   // lfe
            0.3694f, // back
        };
```
Might yield better results as this is what we had before. Is there any benefit to the above constants as opposed to what we had before?

Reply:
Front can't be 0.36 or we're in the same position as before, the front will only get 36% of the output and volume will be stuck low. I got these numbers from googling around, and from section 7.8.2 in this doc: http://www.atsc.org/wp-content/uploads/2015/03/A52-201212-17.pdf

Also -3dB (pow10(-3 / 20)) just seems to be used everywhere, it's in this example on Wiki as well: https://en.wikipedia.org/wiki/Pan_law

Reply:
Fair point, can be kept then. Just wanted to make sure 👍